### PR TITLE
[Upgrade] Nightwatch 1.6.2 / ChromeDriver 91

### PR DESCRIPTION
**Pull request by U-DON**
## Description
In the process of moving Nightwatch tests to our GitHub Actions workflow, I encountered some issues.

The GHA runners by default have ChromeDriver v91 (and may update in the future), which appears to be incompatible with the version of Nightwatch we were using. This caused many Nightwatch tests to break right off the bat when just running `yarn test:e2e:headless` in GHA workflows.

Also, having an outdated ChromeDriver while having a newer version of Chrome meant not being able to run the Nightwatch tests locally for debugging.

### Changes:
- Upgraded Nightwatch. Gets the tests passing with the latest versions of ChromeDriver / Chrome.
- Minor changes in a small number of tests. Required to get them working with the new APIs and commands.
- Made some modifications to the Docker setup to get the tests passing in Jenkins.
    - Switched to using ChromeDriver as a standalone WebDriver without a dependency on Selenium.
    - Removed the `selenium-standalone-server` container from the setup. It was useful for pinning a specific version of Chrome, but it was outdated and incompatible with the latest ChromeDriver.

## Testing done
Passed local testing and CI runs.

## Acceptance criteria
- [ ] Nightwatch tests should pass with the latest ChromeDriver.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -123,59 +123,31 @@ node('vetsgov-general-purpose') {
       // Set timeout of 60 minutes for integration tests
       timeout(60) {
         try {
-          if (commonStages.IS_PROD_BRANCH && commonStages.VAGOV_BUILDTYPES.contains('vagovprod')) {
```
Comment (U-DON) [inline Jenkinsfile:126]: Turns out there was no difference between the branches of this conditional...

File: config/nightwatch.docker-compose.js [hunk 2]
```diff
@@ -19,17 +16,23 @@ module.exports = {
   parallel_process_delay: 10,
   disable_colors: process.env.BUILDTYPE === 'production',
 
+  webdriver: {
+    start_process: true,
+    server_path: process.env.CHROMEDRIVER_FILEPATH || chromedriver.path,
+    port: 9515,
+    status_poll_interval: 1000,
+    max_status_poll_tries: 5,
```
Comment (U-DON) [inline config/nightwatch.docker-compose.js:24]: The default values were 100 and 5, which made the timeout for the ChromeDriver setup a little too strict, causing it to fail before the tests even started.

File: src/applications/claims-status/tests/e2e/04.claim-ask-va.e2e.spec.js [hunk 3]
```diff
@@ -38,23 +38,15 @@ module.exports = E2eHelpers.createE2eTest(client => {
...
   // click on link to page
   client
-    // I have no idea why this pause is required, but it sure is
-    .pause(2000)
     .click('.claims-alert-status a')
     .waitForElementPresent('.usa-button-secondary', Timeouts.normal)
     .axeCheck('.main');
 
-  // click on disabled button
-  client.click('.main .usa-button-primary').pause(500);
```
Comment (U-DON) [inline src/applications/claims-status/tests/e2e/04.claim-ask-va.e2e.spec.js:48]: Clicking on disabled elements will raise errors in this newer version of Nightwatch. Edited instead to ensure that the button is disabled.

**Review by U-DON — COMMENTED** [review]
There's pretty much no reason to keep `selenium-server` as a dependency anymore, but the Nightwatch-Sauce integration still refers to it in its config, so I left it in for now. It would still be straightforward to fully remove it in the future.

**Review by Mottie — APPROVED** [review]

**Review by timwright12 — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (oseasmoran73) [reply Jenkinsfile:126]: I presume the answer is no, but would this still execute if not `PROD_BRANCH`?

**Review by oseasmoran73 — APPROVED** [review]
LGTM

File: Jenkinsfile (hunk 1 above)
Comment (U-DON) [reply Jenkinsfile:126]: This particular conditional? No, but the else block would've been executed, but it seems identical to this if block anyway.

**Pull request merged**